observIQ/bindplane-op
# feat: Make Measurements Interval Configurable

### Proposed Change

- make measurements collection interval configurable on a per config basis

#### Testing

1. Create a config in the ui that will collect some data
2. use the cli to export it to a file using the `--export` flag 
3. Update the config with a new measurement interval:
Ex:
```
spec:
    contentType: ""
    measurementInterval: 60s
```

4. Reapply the config and roll it out
5. watch the server to make sure the collection are coming in at the right interval 

##### Checklist
